.changeset/thirty-timers-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": patch
```

Comment:
Is this a patch or a feature?

Reply:
IMO it's a patch, because it's very small change and so it's not `minor` or `major` bump

Reply:
Ok, thank you, I still need to get used to the naming